src/Lesson5/HomeTask7.java

```diff
@@ -0,0 +1,11 @@
+package Lesson5;
+//Answer the question: What is a package-private access? Create a simple program to demonstrate the package-private access.
+//Tips:
+// - Create two different packages
+// - Create 3 classes. 2 classes should be in the first package, 3rd class - in the second package.
+// - Create in one of the classes some package-private members(fields or methods).
+// - In another two classes create objects of the class with package-private members and try to access its members.
+public class HomeTask7 {
+
+}
```

Comment:
HomeTask7 should be like a demonstration of written task. You have to remove main methods in the Class1, Class2 and create main method here. By the way, I don't see "3rd class - in the second package."